lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -397,6 +398,23 @@ def add_charge_details(post, money, payment, options)
         post
       end
 
+      def add_shipping_address(post, options)
```

Comment:
Also was wondering about your use of `requires!(options, :name)`, I haven't seen requires! used in this way (again clarifying for my own knowledge). Is this just a shorter way of saying `return unless options[:name]` ?

Reply:
Another just personal curiosity question (sorry!): The way that it's written seems like if :name is missing but the other shipping_address info is there it will be ignored/discarded. Is the :name part essential for Stripe's shipping_address?

Reply:
Hi @rachelkirk , I just noticed there is a separate gateway for latest stripe API that is **stripe_payment_intent**, so our changes should go into that gateway specifically. Thanks for pointing out, I will make separate PR for the other gateway (and closing this one).